Data/sync/BraveCryptoJSFunctions.swift

```diff
@@ -0,0 +1,41 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import BraveShared
+
+struct BraveCryptoJSFunctions: BrowserifyExposable, BundledJSProtocol {
```

Comment:
Wondering if this should just be named `BraveCryptoJS` and rename the `Names` struct to simply `Functions`? 

At the very least I don't think this should be plural, right?